mergin/client_pull.py

```diff
@@ -183,8 +203,10 @@ def download_project_is_running(job):
     """
     for future in job.futures:
         if future.done() and future.exception() is not None:
-            _cleanup_failed_download(job.directory, job.mp)
-            raise future.exception()
+            file_path = _cleanup_failed_download(job.directory, job.mp)
+            e = future.exception()
+            e.log_file = file_path
+            raise e
```

Comment:
Hmm adding extra attribute to arbitrary exceptions may not be very nice... How about we instead assign the failed log file to the `job` ? e.g. `job.failure_log_file` (which would be None by default)